.electron-vue/build.js

```diff
@@ -1,156 +0,0 @@
-'use strict'
```

Comment:
So I think this file is just for running `webpack`.
Do we **need** `webpack`?
Do we **want** `webpack`?

Reply:
Yes we need `Webpack`
Vue.js has `webpack` as a dev dependency which implies that `webpack` takes all of you `Vue.js` file and makes one large file for each file type e.g .js .html .css ...